profiles/repositories.py

```diff
@@ -92,5 +96,15 @@ def next_id(self) -> str:
 
         return profile_id
 
+    def list(self, limit: int = None, offset: int = 0, desc: bool = True) -> List[Profile]:
+        query = self.db.session.query(Profile)
+
+        if desc:
+            query = query.order_by(Profile._index_name.desc())  # pylint: disable=protected-access
```

Comment:
There has to be a better way than this?

Reply:
doesn't look like an SQLAlchemy problem as much as accessing a non-public class field. This could be exposed as a static method from the `Profile` class so that only a few selected orderings are available?
```
@staticmethod
def asc():
    return _index_name.asc()

query.order_by(Profile.asc()) # yeah
query.order_by(Profile._other_field.asc()) # you can't
```

Reply:
I thought `query = query.order_by(Profile.name.index.desc())` would work, but get `Neither 'property' object nor 'Comparator' object associated with Profile.name has an attribute 'index'`.

For minute we only order by index name, so that might be ok.

Reply:
@thewilkybarkid could you look over the changes I have committed? as you are the author of the PR I can't select you as a reviewer.